Configuration/StandardSequences/python/DigiToRaw_Repack_cff.py

```diff
@@ -56,3 +56,28 @@
 DigiToHybridRawRepack = cms.Sequence( DigiToHybridRawRepackTask )
 DigiToVirginRawRepack = cms.Sequence( DigiToVirginRawRepackTask )
 DigiToSplitRawRepack = cms.Sequence( DigiToRawRepackTask, DigiToVirginRawRepackTask )
+
+from EventFilter.SiStripRawToDigi.SiStripDigis_cfi import siStripDigis
+siStripDigisHLT = siStripDigis.clone(ProductLabel = "rawDataRepacker")
+
+from RecoLocalTracker.Configuration.RecoLocalTracker_cff import siStripZeroSuppression
```

Comment:
Why not to import it directly from RecoLocalTracker/SiStripZeroSuppression/python/SiStripZeroSuppression_cfi.py?